src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -98,16 +100,25 @@
     };
     const attachFieldDefinitionNodeEvents = (fieldNode, { targetContainer }) => {
         const fieldGroupInput = fieldNode.querySelector('.ibexa-input--field-group');
-        const removeFieldsBtn = fieldNode.querySelectorAll('.ibexa-collapse__extra-action-button--remove-field-definitions');
+        const fieldDefinitionsInputs = fieldNode.querySelectorAll(SELETOR_FIELD_INPUTS_FOR_FIREFOX);
+        const removeFieldBtns = fieldNode.querySelectorAll('.ibexa-collapse__extra-action-button--remove-field-definitions');
         const fieldInputsToValidate = fieldNode.querySelectorAll(SELECTOR_INPUTS_TO_VALIDATE);
         const groupCollapseNode = targetContainer.closest('.ibexa-collapse--field-definitions-group');
         const { fieldsGroupId } = groupCollapseNode.dataset;
 
         fieldInputsToValidate.forEach(attachValidateEvents);
         fieldGroupInput.value = fieldsGroupId;
-        removeFieldsBtn.forEach((removeFieldBtn) => {
+        removeFieldBtns.forEach((removeFieldBtn) => {
             removeFieldBtn.addEventListener('click', removeField, false);
         });
+        fieldDefinitionsInputs.forEach((fieldDefinitionsInput) => {
+            if (!navigator.userAgent.includes('Firefox')) {
```

Comment:
I would check outside of the forEach, there is no reason to check multiple times.